speedy-recovery-client/src/components/secured/shared/Header.js

```diff
@@ -78,6 +87,24 @@ class Header extends Component {
       </header>
     );
   }
+
+  componentDidMount() {
+    if (this.props.role !== "Patient") {
+      let id = this.props.userid;
+      getConversation(id)
+            .then(value => {
+              var unreadNum = 0;
```

Comment:
You shouldnt use var, use let instead. Also change the variable name "value" to smth like conversationResource as  it will be easier to understand  the code.

Reply:
The reason that I was using var in line 96 is I need to change the value of unreadNum in line 98. However, let variable is unchangeable.

Reply:
let is not unchangeable.

Reply:
Yeah, thx for reminding my misunderstanding of the concept!